components/board/customlinux/README.md

```diff
@@ -0,0 +1,17 @@
+## Configuring a custom linux board
+This component supports a board running linux and requires the user to provide a map of gpio pin names to the corresponding gpio chip and line number. The mappings should be provided in a json file in this format:
+```json
+{
+  "*pins": [
+    {
+        "*name": "string",
```

Comment:
just add a comment saying the asterisks are denoting required fields. This will be officially documented in a nicer way when we production it, we don't need to worry this much about this readme, it's for us and whoever wants to use this experimentally